tooling/diff_accepted_roots.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+source source_me.sh
+
+function usage() {
+    echo -e "EXAMPLE:
+    ./$(basename "${0}") [path-to-shard-folder-1] [path-to-shard-folder-2]
+
+    ./$(basename "${0}") oak/2024h1 oak/2024h2
+    ./$(basename "${0}") oak/2023 sapling/202431
```

Comment:
poor sapling, always having 6 digit years
```suggestion
    ./$(basename "${0}") oak/2023 sapling/2024h1
```